packages/ember-views/lib/system/event_dispatcher.js

```diff
@@ -7,11 +7,11 @@ import { assign } from '@ember/polyfills';
 import { assert } from '@ember/debug';
 import { get, set } from 'ember-metal';
 import { Object as EmberObject } from 'ember-runtime';
-import jQuery from './jquery';
+import jQuery, { jQueryDisabled } from './jquery';
 import ActionManager from './action_manager';
 import fallbackViewRegistry from '../compat/fallback-view-registry';
 
-const HAS_JQUERY = jQuery !== undefined;
+const HAS_JQUERY = jQueryDisabled === false;
```

Comment:
I don't think we should check for `false` here (though I suppose it doesn't matter _too_ much).  In general we can trust that our `jQueryDisabled` export will always be boolean and will encapsulate the proper "default to true, unless false" logic.

Reply:
Hmm, any reason to not just use `jQueryDisabled` in this file instead of making the second constant (`HAS_JQUERY`)?